test/framework/alertmanager.go

```diff
@@ -279,8 +278,12 @@ func (f *Framework) GetSilences(ns, n string) ([]amAPISil, error) {
 	return getSilencesResponse.Data, nil
 }
 
+// WaitForAlertmanagerConfigToContainString retrieves the Alertmanager
+// configuration via the Alertmanager's API and checks if it contains the given
+// string.
 func (f *Framework) WaitForAlertmanagerConfigToContainString(ns, amName string, expectedString string) error {
```

Comment:
since all of the parameters are strings, we could simplify the signature as:
```go
func (f *Framework) WaitForAlertmanagerConfigToContainString(ns, amName, expectedString string) error
```